tests/TestingTools/TaskBuilder.ts

```diff
@@ -194,6 +197,20 @@ export class TaskBuilder {
         return this;
     }
 
+    public reminders(reminders: string[]): TaskBuilder {
+        if (reminders.length > 0) {
+            const parsedReminders = new ReminderList(null);
+            for (const reminder of reminders) {
+                const reminderDate = TaskBuilder.parseReminder(reminder);
+                if (reminderDate) {
```

Comment:
I think, after writing other tests, that my statement above is not true, as instead of returning null for an invalid date, it may return a date that gives `Invalid date` as the test.

Either way, it would be good to find a way to catch unintentional errors in date formatting in tests.

Reply:
done